components/tours/crt_tour/crt_list_tutorial_tip.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import {GlobalState} from 'types/store';
+import {getIsMobileView} from 'selectors/views/browser';
+import {ShortcutKey, ShortcutKeyVariant} from 'components/shortcut_key';
+
+import {useMeasurePunchouts} from '@mattermost/components';
+
+import CRTTourTip from './crt_tour_tip';
+
+const translate = {x: -6, y: 62};
+
+export const UpShortcut = () => (
+    <ShortcutKey variant={ShortcutKeyVariant.TutorialTip}>{'UP'}</ShortcutKey>
+);
+
+export const DownShortcut = () => (
+    <ShortcutKey variant={ShortcutKeyVariant.TutorialTip}>{'DOWN'}</ShortcutKey>
+);
+
+const CRTListTutorialTip = () => {
+    const isMobileView = useSelector((state: GlobalState) =>
+        getIsMobileView(state),
+    );
+    const title = (
+        <FormattedMessage
+            id='tutorial_threads.list.title'
+            defaultMessage={'Threads List'}
+        />
+    );
+
+    const screen = (
+        <>
+            <p>
+                <FormattedMessage
```

Comment:
the `FormattedMessage` returns another span that has no function/effect at all in these cases, I'd suggest use just the `intl.formatMessage` when placing text inside `<p>` tags